react/typings/vtex.render-runtime.d.ts

```diff
@@ -1,19 +1,65 @@
 declare module 'vtex.render-runtime' {
   interface Runtime {
     account: string
+    rootPath: string
+    navigate: (args: NavigateArgs) => void
     getSettings(id: string): StoreSettings
     route: {
       routeId: string
     }
   }
 
+  interface NavigateArgs {
+    fallbackToWindowLocation: boolean
+    to: string
+  }
+
   interface StoreSettings {
     storeName: string
     titleTag: string
   }
 
+  interface KeyValue {
+    value: string
+  }
+
+  interface Session {
+    id: string
+    type: undefined
+    namespaces: {
+      store: {
+        channel: string
+      }
+      profile: {
+        isAuthenticated: KeyValue
+        email: string
+      }
+    }
+  }
+
+  interface SessionUnauthorized {
+    type: 'Unauthorized'
+    message: string
+  }
+
+  interface SessionForbidden {
+    type: 'Forbidden'
+    message: string
+  }
+
+  export interface RenderSession {
+    sessionPromise: Promise<SessionPromise>
+  }
+
+  export interface SessionPromise {
+    response: SessionResponse
+  }
+
+  export type SessionResponse = Session | SessionUnauthorized | SessionForbidden
+
   export function useRuntime(): Runtime
   export const Helmet
   export const canUseDOM: boolean
   export const LoadingContextProvider
+  export const Loading
```

Comment:
you can add `export {}` _outside_ the `declare module` block to reuse the typings `vtex.render-runtime` exposes by default, so you don't need to add stuff like this

Reply:
actually, if you do either `import` or `export` outside the declare module, it will be considered as a module and will merge with the thing you are trying to `declare` (I don't know if this explanation is good enough haha)

Reply:
[here is the docs](https://www.typescriptlang.org/docs/handbook/declaration-merging.html) if you want to take a look at it

Reply:
I only removed this one, the others exports are not working so we really need them to fix it.